static/includes/ReplicationIntroSCALE.md

```diff
@@ -3,4 +3,4 @@
 The first snapshot taken for a task creates a full file system snapshot, and all subsequent snapshots taken for that task are incremental to capture differences occurring between the full and subsequent incremental snapshots.
 
 Scheduling options allow users to run replication tasks daily, weekly, monthly, or on a custom schedule. 
-Users also have the option to run a scheduled job on demand. 
+Users also have the option to run a scheduled job on demand, ensuring the high availability (HA) of critical data. 
```

Comment:
High availability (HA) is specific to the systems equipped with the hardware and license to allow it. HA has nothing to do with replication or making replicated data highly available. 
```suggestion
Users also have the option to run a scheduled job on demand. 
```